rollup.config.mjs

```diff
@@ -0,0 +1,20 @@
+import { dts } from 'rollup-plugin-dts'
+
+const config = [
+  {
+    input: 'projects/ngx-meta/dist/json-ld/index.d.ts',
+    output: {
+      format: 'es',
+      file: 'projects/ngx-meta/dist/json-ld/bundled.d.ts',
+    },
+    plugins: [
+      dts({
+        tsconfig: './tsconfig.rollup.json',
+        respectExternal: true,
+      }),
+    ],
+    external: [/node_modules/, /ngx-meta\/dist\/(?!json-ld)/],
```

Comment:
Ignore references to other `ngx-meta` entry points, those definitions will be bundled as part of those entry points